src/frontend/apps/drive/src/features/explorer/components/tree/ExplorerTree.tsx

```diff
@@ -168,10 +181,57 @@ export const ExplorerTree = () => {
           selectedNodeId={itemId}
           initialOpenState={initialOpenState}
           afterMove={handleMove}
+          beforeMove={(moveResult, moveCallback) => {
+            if (!moveResult.newParentId || !moveResult.oldParentId) {
```

Comment:
Might be great to explain how this can happens, when reading this code I was wondering from which user interaction the condition is protecting the function. 🤔

( Appart from TS compilation, but it's not the point here ) - or add this in the todo regarding to doc of the tree in the ui-kit.